shopBuddy/shopBuddy/AddItemViewController.m

```diff
@@ -6,29 +6,27 @@
 //
 
 #import "AddItemViewController.h"
-#import "APIManager.h"
+#import "BarcodeAPIManager.h"
 #import "Item.h"
 #import "ItemDetailViewController.h"
 #import "ScanBarcodeViewController.h"
 #import "SearchResultItemCell.h"
 #import "UIImageView+AFNetworking.h"
 
 @interface AddItemViewController () <UITableViewDataSource, UITableViewDelegate>
-{
-    UITableView *tableView;
-    UITextField *barcodeField;
-    UITextField *searchField;
-    NSMutableArray<Item*> *searchResults;
-}
+@property UITextField *searchField;
+@property NSMutableArray<Item*> *searchResults;
+@property UITextField *barcodeField;
+@property UITableView *tableView;
```

Comment:
properties are slower than ivars, you only should use properties if you need to add any attributes like atomic, etc.

You can move these to the @implementation block below and dot notation should work.
```
@implementation AddItemViewController {
    UITableView *_tableView;
    UITextField *_barcodeField;
    UITextField *_searchField;
    NSMutableArray<Item*> *_searchResults;
}

...
```